spec/Dynamic-Validator.md

```diff
@@ -251,7 +251,7 @@ To break a tie, we give priority to the candidate who have responded most recent
 Current validators will have highest priority among candidates with the same `(delegation, deposit)` at the next election.
 The sender of most recent `SelfNominate` transaction will be after them.
 
-Thus, We repriortize candidates on `SelfNomination` transaction and `TermEnd` event with the following algorithm.
+Thus, we reprioritize candidates on `SelfNomination` transaction with the following algorithm.
```

Comment:
Thus, we reprioritize candidates on the `SelfNomination` transaction with the following algorithm.